site/docs/v1.3.1/hooks.md

```diff
@@ -75,7 +75,15 @@ velero backup get nginx-hook-test
 velero backup logs nginx-hook-test | grep hookCommand
 ```
 
+## Using Multiple Commands
+
+To use multiple commands, wrap your target command in a shell and separate them with `;`, `&&`, or other shell conditional constructs.
+
+```shell
+    pre.hook.backup.velero.io/command='["/bin/bash", "-c", "echo hello > hello.txt && echo goodbye > goodbye.txt"]'
+```
+
 
 [1]: api-types/backup.md
-[2]: https://github.com/vmware-tanzu/velero/blob/v1.3.1/examples/nginx-app/with-pv.yaml
+[2]: https://github.com/vmware-tanzu/velero/blob/master/examples/nginx-app/with-pv.yaml
```

Comment:
Is this change intentional?
We might want to keep the examples also versioned, just in case we update this example in master.

Reply:
It was not...I just copied the file directly over, will undo it.